src/SecureElementReader.App/Views/MainWindow.axaml

```diff
@@ -4,59 +4,87 @@
+			<Setter Property="CornerRadius" Value="0"/>
+			<Setter Property="BorderThickness" Value="0"/>
+			<Setter Property="Margin" Value="5"/>
+			<Setter Property="HorizontalAlignment" Value="Stretch"/>
+			<Setter Property="HorizontalContentAlignment" Value="Center"/>
+			<Setter Property="VerticalContentAlignment" Value="Center"/>
+			<Setter Property="Foreground" Value="White"/>
+		</Style>
+
+		<Style Selector="TextBlock.h2">
+			<Setter Property="VerticalAlignment" Value="Center"/>
+		</Style>
+
+		<Style Selector="TextBox.h3">
+			<Setter Property="Margin" Value="6"/>
+			<Setter Property="TextWrapping" Value="Wrap"/>
+			<Setter Property="CornerRadius" Value="0"/>
+			<Setter Property="BorderThickness" Value="0"/>
+			<Setter Property="IsEnabled" Value="False"/>
+		</Style>
+	</Window.Styles>
+
+
+	<Grid x:Name="BaseWindowGrid" Width="800" Height="600">
+
+		
+		<Grid x:Name="DefinedGrid"
+			  ColumnDefinitions="100,100,100,100,100,100,100,100" RowDefinitions="auto,auto,auto,auto,auto,auto,auto">
+			
+			
+			<DockPanel x:Name="Titlebar" Grid.Column="0" Grid.ColumnSpan="8" Grid.Row="0" Height="30" Width="800">
+				<titlebars:WindowsTitleBar IsSeamless="True"></titlebars:WindowsTitleBar>
+				<titlebars:MacosTitleBar IsSeamless="False"></titlebars:MacosTitleBar>
+			</DockPanel>
+			
+			<TextBlock Grid.Column="0" Grid.Row="1" Grid.ColumnSpan="8" Width="800" Height="25"
+								TextAlignment="Center" Text="{DynamicResource Welcome}" FontSize="20" Foreground="#5b6372" />
+			
+			<Rectangle Grid.Column="2" Grid.Row="2" Grid.ColumnSpan="4" 
+						HorizontalAlignment="Center" Width="500" Fill="Black" Height="2"/>
+			
+			<Button Grid.Column="0" Grid.Row="3" Grid.ColumnSpan="2" Classes="B"
+							Command="{Binding RefreshReaderListCommand}" Background="#19aa8c" Content="{DynamicResource GetReader}"/>
+			 
+			<TextBox Grid.Column="2" Grid.Row="3" Grid.ColumnSpan="3" Margin="5" CornerRadius="0" BorderThickness="0" Foreground="Black"
+					 IsEnabled="False" Text="{Binding CardReaderName}" Watermark="{DynamicResource NoCardReadesFounded}" HorizontalContentAlignment="Center" VerticalContentAlignment="Center"/>
+
+			<Button Grid.Column="0" Grid.Row="4" Grid.ColumnSpan="2" Classes="B"
+					Command="{Binding CertDetailsCommand}" Background="#21bcbe"  Content="{DynamicResource GetCert}" IsEnabled="{Binding IsEnabled}"/>
+
+			<Button Grid.Column="2" Grid.Row="4" Grid.ColumnSpan="2" Classes="B"
+					Command="{Binding VerifyPinCommand}" Background="#21bcbe"  Content="{DynamicResource VerifyPin}" IsEnabled="{Binding IsEnabled}"/>
+
+			<Image Grid.Column="6" Grid.Row="1" Grid.ColumnSpan="2" Grid.RowSpan="4" HorizontalAlignment="Right"
+					Width="100" Height="100" Margin="0,0,25,0" Source="/Assets/taxcore.png"/>
+
+			<main:CertDetailsView Grid.Column="0" Grid.Row="5" Grid.ColumnSpan="8"					
+					DataContext="{Binding CertDetailsViewModel}"/>
+
+			<Grid x:Name="OverlayGrid" Grid.Row="0" Grid.RowSpan="6" Grid.Column="0" Grid.ColumnSpan="8">
```

Comment:
Overlay need to be over all window, which is not case now
![image](https://user-images.githubusercontent.com/68274836/199219095-717ccb3c-0f32-4000-b3af-64bcff8bff9e.png)